packages/union-component/src/components/DonationPaymentForm.tsx

```diff
@@ -153,7 +178,7 @@ const DonationPaymentForm: React.FC<Props> = ({
           required
           title="Contact phone number"
         />
-        {hasChapterSelection && (
+        {hasChapterSelection ? (
```

Comment:
It is "bad" to use `&&`? Just for me to be aware for future code.

Reply:
Makes sense to me. Thanks for sharing. I don't like ternaries but it is a excellent explanation over there